linux_os/guide/system/permissions/partitions/mount_option_boot_nosuid/rule.yml

```diff
@@ -0,0 +1,26 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Add nosuid Option to /boot'
+
+description: |-
+    The <tt>nosuid</tt> mount option can be used to prevent
+    execution of setuid programs in <tt>/boot</tt>. The SUID and SGID permissions
+    should not be required in these user data directories.
```

Comment:
boot partition shouldn't contain user data.

Reply:
Fixed in 22122d07991cf6d5f74e963c91d12913b9c425f9.